posts/first-lean-release.md

```diff
@@ -0,0 +1,49 @@
+---
+author: 'Scott Morrison'
+category: 'announcement'
+date: 2023-08-29 16:30:00 UTC+02:00
```

Comment:
Are you now in europe? Or was this written in Leiden?

Reply:
```suggestion
date: 2023-09-08 15:58:40 UTC+10:00
```